### PR TITLE
Container Team minutes 2023-02-06

### PR DESCRIPTION
**Pull request by jschoone**

### CODE REVIEW

File: container/20230206.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Community call report and minutes transfer
+  @jschoone
+
+## Agenda
+See Kanban board 
+
+### Ingress and Loadbalancing (issues/#227)
+* issues/#251: octavia ovn provider appears to be providing wanted L3 LB functionality
+- Available on gx-scs, pco, not on wavestack (ovn not used)
```
Comment (joshmue) [inline container/20230206.md:25]: Indentation is lost. Using the etherpad's export feature (as text file) should help a bit.

**Comment by joshmue** [comment]
As the DCO bot/check says: A sign-off is missing.

File: container/20230206.md (hunk 1 above)
Comment (fkr) [reply container/20230206.md:25]: @joshmue if you look at the branch, one sees the commit that I did last night to fix the formatting. No clue why it does not show up here.

**Review by fkr — APPROVED** [review]
LGTM.

**Pull request merged**